.github/workflows/build_test.yml

```diff
@@ -7,6 +7,30 @@ on:
     branches: [main]
 
 jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check License Header
+        uses: apache/skywalking-eyes/header@main
+        with:
+          header:
+            license:
+              spdx-id: Apache-2.0
+      - name: Check CMake files
+        id: cmake-format
+        uses: PuneetMatharu/cmake-format-lint-action@v1.0.0
```

Comment:
I am hesitant to rely on a 3rd party project this small.  The actual command is `find . \( -name '*.cmake' -o -name 'CMakeLists.txt' \) -exec cmake-format $* {} +`.  Can we maybe use that directly?

Reply:
Updated